packages/instrumentation-typeorm/test/Repository.spec.ts

```diff
@@ -24,9 +24,8 @@ describe('Repository', () => {
         await repo.insert(user);
         const [users, count] = await repo.findAndCount();
         expect(count).toBeGreaterThan(0);
-
         const spans = getTestSpans();
-        expect(spans.length).toEqual(1);
+        expect(spans.length).toEqual(2);
```

Comment:
why did this change?

Reply:
not sure how it worked before but this is the expected behavior since we do 2 operations 
```
  await repo.insert(user);
  const [users, count] = await repo.findAndCount();
```